samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -43,7 +43,7 @@ private void Update()
         if (mapComponent != null && mapComponent.APIKey == "")
         {
             mapComponent.APIKey = APIKey;
-#if (UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX)
+#if (UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX || UNITY_IOS)
```

Comment:
This is still an issue? If so can you make an issue in SSV?

Reply:
Yes it is still an issue, I will make one in SSV

Reply:
made: https://devtopia.esri.com/runtime/ssv-normandy/issues/1519